python/TestHarness/TestHarness.py

```diff
@@ -972,6 +972,11 @@ def useExistingStorage(self):
         if (os.path.exists(self.options.results_file)
             and (self.options.failed_tests or self.options.pbs or self.options.show_last_run)):
             return True
+        elif ((self.options.failed_tests or self.options.show_last_run)
```

Comment:
I wasn't sure what to do about PBS... I'd rather not mess with that logic. I _believe_ we are generating one for the user if it does not exist and they are launching PBS jobs for the first time...